src/QuadForm/Quad/ZLattices.jl

```diff
@@ -646,3 +652,98 @@ function maximal_integral_lattice(L::ZLat)
   M = maximal_integral_lattice(LL)
   return _to_ZLat(M, V = ambient_space(L))
 end
+
+################################################################################
+#
+#  Rational equivalent
+#
+################################################################################
+
+@doc Markdown.doc"""
+    isrationally_equivalent(L::ZLat, M::ZLat, p) -> Bool
+
+Returns whether the rational spans of $L$ and $M$ are equivalent over the
+completion at $p$.
+"""
+function isrationally_equivalent(L::ZLat, M::ZLat, p)
+  return isequivalent(rational_span(L), rational_span(M), p)
+end
+
+@doc Markdown.doc"""
+    isrationally_equivalent(L::ZLat, M::ZLat)-> Bool
+
+Returns whether the rational spans of $L$ and $M$ are equivalent.
+"""
+function isrationally_equivalent(L::ZLat, M::ZLat)
+  return isequivalent(rational_span(L), rational_span(M))
+end
```

Comment:
This can be removed, as this is already covered by the functions in https://github.com/thofma/Hecke.jl/blob/master/src/QuadForm/Lattices.jl